src/PrestaShopBundle/Command/ShopCreatorCommand.php

```diff
@@ -0,0 +1,247 @@
+            $order->total_products = $cart->getOrderTotal(false, Cart::ONLY_PRODUCTS);
+            $order->total_products_wt = $cart->getOrderTotal(true, Cart::ONLY_PRODUCTS);
+            $order->total_shipping = 0;
+            $order->secure_key = md5('test');
+            $order->carrier_tax_rate = 0;
+            $order->conversion_rate = 1;
+
+            $order->add();
+
+            $orderStatus = new OrderState(3); // 3 est l'ID du statut de la commande "en cours de préparation"
+            $order->setCurrentState($orderStatus->id);
+            $order->save();
+
+            $output->writeln('Order ' . ($i + 1) . ' created successfully.');
+        }
+
+        for ($i = 0; $i < $numberOfCartRules; ++$i) {
+            $cartRule = new CartRule();
+            $cartRule->name = [$idLang => 'Fake Cart Rule ' . ($i + 1)];
+            $cartRule->description = 'Generated by the Faker library';
+            $cartRule->code = strtoupper($faker->bothify('FAKE-CART-RULE-????'));
+            $cartRule->quantity = $faker->numberBetween(1, 100);
+            $cartRule->quantity_per_user = $faker->numberBetween(1, 10);
+            $cartRule->priority = $faker->numberBetween(1, 10);
+            $cartRule->partial_use = $faker->boolean(50);
+            $cartRule->minimum_amount = $faker->randomFloat(2, 0, 500);
+            $cartRule->minimum_amount_currency = 1; // Change this value according to the currencies available in your store
+            $cartRule->reduction_percent = $faker->numberBetween(10, 90);
+            $cartRule->free_shipping = $faker->boolean(50);
+            $cartRule->active = true;
+
+            // Set the date range for the cart rule
+            $startDate = $faker->dateTimeBetween('-1 year', 'now');
+            $endDate = $faker->dateTimeBetween($startDate, '+1 year');
+            $cartRule->date_from = $startDate->format('Y-m-d H:i:s');
+            $cartRule->date_to = $endDate->format('Y-m-d H:i:s');
+
+            $cartRule->add();
+            $output->writeln('Cart Rule ' . ($i + 1) . " created successfully.\n");
+        }
+
+        for ($i = 0; $i < $numberOfProducts; ++$i) {
+            $product = new Product();
+            $productName = 'Fake Product ' . ($i + 1);
+
+            $product->name = [$idLang => $productName];
+            $product->description = 'This is a fake product generated by the Faker library.';
+            $product->description_short = 'Generated by the Faker library.';
+            $product->price = $faker->randomFloat(2, 1, 1000);
+            $product->reference = $faker->bothify('FAKE-PRODUCT-????');
+            $product->active = true;
+
+            $product->add();
+
+            // Associate the product to categories (use your own category IDs)
+            $categories = [2, 3, 4];
+            $product->updateCategories($categories);
+
+            echo 'Product ' . ($i + 1) . " created successfully.\n";
```

Comment:
At the end of the process you could also start indexation of the products for Search